frontend/package-lock.json

```diff
@@ -4493,9 +4493,9 @@
       }
     },
     "node_modules/@oddbird/popover-polyfill": {
-      "version": "0.2.3",
-      "resolved": "https://registry.npmjs.org/@oddbird/popover-polyfill/-/popover-polyfill-0.2.3.tgz",
-      "integrity": "sha512-XDK+V/gUeE4NEsWp79eVzhlK3wuVcRDJuaas63qo0IJLJpyOLHqycJLFYvuq8kebgT1nl87P3sbSb5ZN6Vyf5g=="
+      "version": "0.3.2",
```

Comment:
@HDinger are these updates OK?

Reply:
Yes. Primer has updated its depedencies for `popover-polyfill` so we have to update as well